core/services/vrf/delegate_test.go

```diff
@@ -300,15 +302,15 @@ func TestDelegate_ValidLog(t *testing.T) {
 		// Ensure we queue up a valid eth transaction
 		// Linked to requestID
 		vuni.txm.On("CreateEthTransaction",
-			mock.MatchedBy(func(newTx txmgr.NewTx) bool {
+			mock.MatchedBy(func(newTx txmgr.EvmNewTx) bool {
 				meta := newTx.Meta
-				return newTx.FromAddress == vuni.submitter &&
-					newTx.ToAddress == common.HexToAddress(jb.VRFSpec.CoordinatorAddress.String()) &&
+				return newTx.FromAddress.String() == vuni.submitter.String() &&
+					newTx.ToAddress.String() == common.HexToAddress(jb.VRFSpec.CoordinatorAddress.String()).String() &&
```

Comment:
Is there no more efficient way to compare these?

Reply:
Done, simplfiied based on the new type.